content/rs/concepts/shard-placement-policy.md

```diff
@@ -0,0 +1,53 @@
+---
+Title: Shard Placement Policy
+description:
+weight: $weight
+alwaysopen: false
+categories: ["RS"]
+aliases: /rs/concepts/rebalancing-shard-placement/
+---
+{{< embed-md "shard-placement-intro.md"  >}}
+
+## Shard Placement Policy
+
+The shard placement policies are:
+
+- dense - Place as many shards as possible on the smallest number of nodes to reduce the latency between the proxy and the database shards.
+- sparse - Spread the shards across as many nodes in the cluster as possible to spread the traffic across cluster nodes.
+
+When you create a cluster, the default shard placement policy on the cluster is **dense**.
+The default policy is assigned to all databases that you create on the cluster.
+
+### Dense Shard Placement Policy
+
+In the dense policy, shards are placed on the same node
```

Comment:
This still seems unclear. I don't think that a reader will understand what "shards are placed on the same node" means. I have a hard parsing that idea. I think it's more accurate to say that "all shards of a given type are placed on as few nodes as possible, as long as these nodes have enough free memory to support the associated database configuration. In a dense configurations, shards will be distributed across a minimum of two nodes, where all master shards reside on the first node and and slave shards reside on the second node."

Reply:
I'd also suggest using the active voice here. So "...the cluster places all shards of a given type..." instead of "all shards of a given type are placed..."

Reply:
Lots of good feedback here. We don't define "shard type" so I wold avoid that. Maybe we can simplify with: In the dense policy, the cluster places the database shards on as few nodes as possible, as long as the nodes have enough resources to support the database configuration.

We could add the second sentence but is it specific to dense, or is it the same for sparse: Shards are distributed across a minimum of two nodes, so that all master shards reside on the first node and all slave shards reside on the second node.

Reply:
I suggest to say something like 'the cluster tries to place the database shards on as few nodes as possible' (and similarly for sparse), since the 'enough resources' is only part of the story, as shown in RED-33481.